### PR TITLE
Docs refactor

### PR DESCRIPTION
**Pull request by marcoow**
This combines several improvements around documentation.

## TODO

- [ ] ~~I think we should rename the `session-account` service in the dummy app to `current-user` as well (and its methods and properties) so that it matches the example in the guide. Otherwise the 2 implementation might lead to confusion as they'd use different naming for essentially the same things making it harder to see that both are actually the same.~~ doesn't really work as the model is called `Account` instead of `User` so that would probably be even more confusing.

### CODE REVIEW

**Comment by marcoow** [comment]
@kylemellander: can you review again?

File: guides/managing-current-user.md [hunk 1]
```diff
@@ -23,24 +61,19 @@ const { inject: { service }, isEmpty, RSVP } = Ember;
...
 
-  account: null,
-
   loadCurrentUser() {
-    return new RSVP.Promise((resolve, reject) => {
-      return this.get('store').find('user', 'me').then(
-        user => resolve(this.set('user', user)),
-        reject
-      );
+    return this.get('store').find('user', 'me').then((account) => {
```
Comment (kylemellander) [inline guides/managing-current-user.md]: maybe we should keep the user variable name here to keep it consistent with the find action to keep things clear.

**Comment by kylemellander** [comment]
@marcoow - Just one little thing with variable names, but overall, solid.

File: guides/managing-current-user.md (hunk 1 above)
Comment (marcoow) [inline guides/managing-current-user.md]: Yeah, makes sense - I'll rename the service to `currentUser` or so as well.

**Comment by marcoow** [comment]
@kylemellander: fixed

**Comment by marcoow** [comment]
oops - wrong button

**Pull request merged**